samples/notebooks/Lightweight Python components - sharing 1 - produce component files.ipynb

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
It seems neither notebook is big --- merge them? How to produce and how to load are very related.